project/BuildPlugin.scala

```diff
@@ -243,7 +251,7 @@ object BuildImplementation {
 
     private val NewLine = System.lineSeparator
     def scriptedSettings(testDirectory: sbt.SettingKey[File]): Seq[Def.Setting[_]] = List(
-      ScriptedKeys.scriptedBufferLog := false,
+      ScriptedKeys.scriptedBufferLog := true,
```

Comment:
Can we disable it or is it required?

Reply:
It's required, otherwise we log too much and the job is killed by Drone.